modules/administration-guide/pages/configuring-storage-sizes.adoc

```diff
@@ -57,4 +57,14 @@ spec:
 <2> Specify a claim size on the next line or omit the next line to set the default claim size value. The specified claim size is only used when you select this storage strategy.
 <3> The claim size must be specified as a link:https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. The available quantity units include: `Ei`, `Pi`, `Ti`, `Gi`,  `Mi` and `Ki`.
 
-pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
\ No newline at end of file
+pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
+
+[IMPORTANT]
+====
+Manually modifying a PVC on the cluster that was provisioned by {prod-short} is not officially supported and may have unexpected consequences.
+
+If you want to resize a PVC, you have two options:
+
+* Modify the devfile's volume sizes accordingly and restart the workspace.
```

Comment:
I actually think the devworkspace might have to be deleted and recreated (not just restarted) :( I'll verify this in a bit and get back to you.

Reply:
I just verified this, and you have to modify the devfile's volume sizes accordingly (and backup these changes to git or wherever your devfile is stored) and **delete the workspace, and re-create the workspace** from your modified devfile. 

Restarting the workspace is not sufficient to change the PVC size. This is a feature I'd like to implement though, I hope it can become priortized :')

Reply:
since you have to delete the workspace and re-create it, should we even mention it as an option? creating a brand new workspace with a different PVC size seems a little bit faster.

Reply:
Agreed -- it doesn't make sense to suggest deleting and re-creating the workspace. 

>  creating a brand new workspace with a different PVC size seems a little bit faster.

Agreed :+1: